assets/js/services/helpers.test.js

```diff
@@ -0,0 +1,75 @@
+import {
+  buildImageURL,
+  escapeDoubleQuotes,
+  formatDate,
+  getClassFromIngestSheetStatus
+} from "./helpers";
+
+describe("Build Image URL function", () => {
+  it("should return URL", () => {
+    const expected =
+      "http://localhost:8183/iiif/2/filesetId/square/500,500/0/default.jpg";
+    const actual = buildImageURL("filesetId", "IIIF_SQUARE");
```

Comment:
This totally works fine, but if I was quickly scanning this code I may confuse "filesetId" for representing a variable value.  When passing around mock data in tests, make it something like the real data you're mocking, but more simple.   How about something like:

`const actual = buildImageURL("ABC123", "IIIF_SQUARE");` ?

Reply:
Ok, I'll change it to reflect close to real value.